src/qml/pages/wallet/RequestPayment.qml

```diff
@@ -0,0 +1,239 @@
+
+                Separator {
+                    Layout.fillWidth: true
+                }
+
+                LabeledTextInput {
+                    id: message
+                    Layout.fillWidth: true
+                    labelText: qsTr("Message")
+                    placeholderText: qsTr("Enter message...")
+                }
+
+                Separator {
+                    Layout.fillWidth: true
+                }
+
+                Item {
+                    Layout.fillWidth: true
+                    Layout.minimumHeight: addressLabel.height + copyLabel.height
+                    Layout.topMargin: 10
+                    height: addressLabel.height + copyLabel.height
+                    CoreText {
+                        id: addressLabel
+                        anchors.left: parent.left
+                        anchors.top: parent.top
+                        horizontalAlignment: Text.AlignLeft
+                        width: 110
+                        text: qsTr("Address")
+                        font.pixelSize: 18
+                        color: Theme.color.neutral9
+                    }
+                    CoreText {
+                        id: copyLabel
+                        anchors.left: parent.left
+                        anchors.top: addressLabel.bottom
+                        horizontalAlignment: Text.AlignLeft
+                        width: 110
+                        text: qsTr("copy")
+                        font.pixelSize: 18
+                        color: Theme.color.neutral7
+                    }
+
+                    Rectangle {
+                        anchors.left: addressLabel.right
+                        anchors.right: parent.right
+                        anchors.top: parent.top
+                        anchors.bottom: parent.bottom
+                        color: Theme.color.neutral2
+                        radius: 5
+                        CoreText {
+                            id: address
+                            anchors.fill: parent
+                            anchors.leftMargin: 5
+                            horizontalAlignment: Text.AlignLeft
+                            font.pixelSize: 18
+                            color: Theme.color.neutral9
+                            wrap: true
+                        }
+                    }
+                }
+
+                ContinueButton {
+                    id: continueButton
+                    Layout.fillWidth: true
+                    Layout.topMargin: 30
+                    text: qsTr("Create bitcoin address")
+                    onClicked: {
+                        if (!clearRequest.visible) {
+                            requestCounter = requestCounter + 1
+                            clearRequest.visible = true
+                            title.text = qsTr("Payment request #" + requestCounter)
+                            address.text = "bc1q f5xe y2tf 89k9 zy6k gnru wszy 5fsa truy 9te1 bu"
```

Comment:
Not likely, and inadvisable behavior, but it seems to me like a bad idea to put a placeholder address here when someone curious might build this branch and try to use it with mainnet funds, maybe insert an invalid character or something else to indicate this is a placeholder?

Reply:
cc @GBKS